services/neo4j/init/neo4j-queries.cql

```diff
@@ -37,3 +37,4 @@ CREATE CONSTRAINT ON (`connected_device`:`connected_device`) ASSERT `connected_d
 CREATE INDEX ON :`connected_device` (`flow_id`);
 CREATE INDEX ON :`connected_device` (`source`);
 CREATE INDEX ON :`connected_device` (`mac_address`);
+CREATE INDEX ON :`connected_device` (`type`);
```

Comment:
this shouldn't be here, since it's part of 2698

Reply:
it's not here